util/util.go

```diff
@@ -172,3 +173,27 @@ func Backtrace(size int) []string {
 
 	return str[0:count]
 }
+
+// HistoryFilePath returns the path of the history file
+// $HOME/.local/.faktory-cli.history
+// if the .local folder does not exists, it will create it.
+func HistoryFilePath() string {
+	usr, _ := user.Current()
```

Comment:
This code should stay in the cmd package. Util is for reusable code across several packages.